internal/services/cdn/cdn_endpoint_custom_domain_resource.go

```diff
@@ -168,6 +168,8 @@ func resourceArmCdnEndpointCustomDomainCreate(d *pluginsdk.ResourceData, meta in
 		return fmt.Errorf("waiting for creation of %q: %+v", id, err)
 	}
 
+	d.SetId(id.ID())
```

Comment:
presumably based on #17377, we'll also want to bump the timeouts for this resource?

Reply:
The timeout is from the user's pipeline, this resource has already been set with a quite high timeout value (12 hours).